runtime/gc_realtime/IncrementalOverflow.cpp

```diff
@@ -211,25 +209,27 @@ MM_IncrementalOverflow::fillFromOverflow(MM_EnvironmentBase *env, MM_Packet *pac
 	MM_EnvironmentRealtime *envRealtime = MM_EnvironmentRealtime::getEnvironment(env);
 	MM_HeapRegionDescriptorRealtime* region;
 	MM_RealtimeGC *realtimeGC = envRealtime->getExtensions()->realtimeGC;
+#if defined(OMR_GC_ARRAYLETS)
```

Comment:
I believe this is not correct. How `realtimeGC->getMarkingScheme()` is related with fact that Arraylets are enabled?

Reply:
I see what it is. Sorry, my mistake. It is ok